### ISSUE TITLE
Feature request: generate test by module/function

### ISSUE THREAD

**Issue by yadimon** [rubberduck-vba/Rubberduck#751]
it would be nice to let the RD to generate new test module by selected one. (maybe on right click on a module as option too)
Maybe with already predefined test names, depends on module methods.

**Comment by retailcoder** [comment]
Let's generate a test method stub in a new test module, for each public member of the selected class/module. This will need a little bit of restructuring in the unit test templating.

**Comment by rkapka** [comment]
I'm grabbing this one.

**Comment by rkapka** [comment]
I have some questions related to this task:

- It is already possible to right-click a module and select Add Module -> Test Module from the Code Explorer window. Do we also want to include this option when right-clicking a module in the native VBE?
- Do we want to change the current behaviour of creating a test module for a specific module in the Code Explorer? Or add another option - _Test module with stubs_?

**Comment by retailcoder** [comment]
Can't recall if we added the "Add test module" command to the VBE's *Project Explorer*, but if we did, then yeah; if we didn't, then it can't hurt to add it.

The current "Add test module" command shouldn't change at all - it's possible to invoke it from the main RD menu, and that command has been around since v1.0... I wouldn't change its behavior.

IOW it should be a new command, e.g. "Generate tests for '{moduleName}'", generating stubs for all `Public` (implicit or explicit) members of that module/class.
